### PR TITLE
session_test: move out some tests.

### PR DESCRIPTION
**Pull request by hanfei1991**
Some plan or aggregation test have been covered in other tests, so we don't need them any more.

### CODE REVIEW

File: session_test.go [hunk 1]
```diff
@@ -2133,53 +2132,6 @@ func (s *testSessionSuite) TestSkipWithGrant(c *C) {
...
-
-	sql := "select c1 from t where c1 in (1) and c2 < 10"
-	checkPlan(c, se, sql, "Index(t.idx_c1_c2)[[1 -inf,1 10)]->Projection")
-	mustExecMatch(c, se, sql, [][]interface{}{{1}})
-
-	sql = "select c1 from t where c1 in (1) and c2 > 3"
-	checkPlan(c, se, sql, "Index(t.idx_c1_c2)[(1 3,1 +inf]]->Projection")
-	mustExecMatch(c, se, sql, [][]interface{}{{1}})
-
-	// TODO: c2 is int which will be added cast to real when building LT, thus we cannot extract access condition for it.
```
Comment (hanfei1991) [inline session_test.go]: This problem has been resolved

**Comment by coocood** [comment]
Please provide the links to the test cases that cover the removed ones.

**Comment by hanfei1991** [comment]
agg + index https://github.com/pingcap/tidb/blob/master/executor/aggregate_test.go#L231
agg + having https://github.com/pingcap/tidb/blob/master/executor/aggregate_test.go#L136
test index https://github.com/pingcap/tidb/blob/master/executor/executor_test.go#L991
@coocood

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**